spec/models/inventory_spec.rb

```diff
@@ -1,107 +1,116 @@
 require 'rails_helper'
 include InventoryExceptions
-
-RSpec.describe Inventory, type: :model do
-  before :all do # this is only for the mocks!
-    @uuid = 'not real uuid'
+# these tests must be run in order and all at once
+RSpec.describe Inventory, order: :defined, type: :model do
+  let(:name) do
+    name = ''
+    name += (rand(26) + 65).chr until name.length > 120
+    name
   end
 
-  it 'can raise an exception' do
-    expect { Inventory.mock_exception }.to raise_error(InventoryError)
+  before :each do
+    Timecop.freeze
```

Comment:
One of the methods in the inventory model searches through reservations
between a time. But I really isnt necessary now that you mention it because
it depends on the remote server as well.
On Apr 19, 2016 1:32 PM, "David Faulkenberry" <notifications@github.com>
wrote:

> In spec/models/inventory_spec.rb
> <https://github.com/umts/probable-engine/pull/48#discussion_r60274452>:
>
> >    end
> >
> > -  it 'can raise an exception' do
> > -    expect { Inventory.mock_exception }.to raise_error(InventoryError)
> > +  before :each do
> > +    Timecop.freeze
>
> Out of curiosity, why is this needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/umts/probable-engine/pull/48/files/ff532da7b9d3632ab2ab5cff73e8dcf0c4bc0d20#r60274452>
>